--- pycares-pycares-3.0.0/docs/conf.py.orig	2019-08-24 16:06:57.411276212 +0200
+++ pycares-pycares-3.0.0/docs/conf.py	2019-08-24 16:07:26.171598625 +0200
@@ -14,7 +14,7 @@
 import re
 
 def get_version():
-    return re.search(r"""__version__\s+=\s+(?P<quote>['"])(?P<version>.+?)(?P=quote)""", open('../pycares/_version.py').read()).group('version')
+    return re.search(r"""__version__\s+=\s+(?P<quote>['"])(?P<version>.+?)(?P=quote)""", open('../src/pycares/_version.py').read()).group('version')
 _version = get_version()
 
 # If extensions (or modules to document with autodoc) are in another directory,
