diff --git a/Cython/Utility/ModuleSetupCode.c b/Cython/Utility/ModuleSetupCode.c
index 2aeabd2..8b285bb 100644
--- a/Cython/Utility/ModuleSetupCode.c
+++ b/Cython/Utility/ModuleSetupCode.c
@@ -669,7 +669,7 @@ static CYTHON_INLINE void * PyThread_tss_get(Py_tss_t *key) {
 #if defined(__clang__)
     #define CYTHON_SMALL_CODE
 #elif defined(__GNUC__)
-    #define CYTHON_SMALL_CODE __attribute__((optimize("Os")))
+    #define CYTHON_SMALL_CODE __attribute__((cold))
 #else
     #define CYTHON_SMALL_CODE
 #endif
